packages/code-du-travail-frontend/src/search/SearchResults/index.js

```diff
@@ -14,7 +14,9 @@ const SearchResults = ({
   isSearch,
   query
 }) => {
-  const router = useRouter();
```

Comment:
we can remove useRouter but we need to verify that we call SearchResult with a correct `query` props or no query